modules/touch/src/win32_touch.cpp

```diff
@@ -155,27 +214,82 @@ Win32TouchHook::Win32TouchHook(void* nativeWindow)
 
     if (!gStarted) {
         gStarted = true;
+        gStartTime = std::chrono::duration_cast<std::chrono::microseconds>(
+                            std::chrono::high_resolution_clock::now().time_since_epoch());
+#ifdef ENABLE_TUIOMESSAGES
         gTuioServer = new TUIO::TuioServer("localhost", 3333);
         TUIO::TuioTime::initSession();
+#endif
         gTouchHook = SetWindowsHookExW(
             WH_GETMESSAGE,
             HookCallback,
             GetModuleHandleW(NULL),
             GetCurrentThreadId()
         );
+
+        // In theory, if our UI is pumped from a different thread, we can
```

Comment:
Consider also moving this code suggestion to an issue instead.

Reply:
Opened an issue, see #1042